ChangeLog

```diff
@@ -1,3 +1,4 @@
+	* fix cxxflags for python binding build
```

Comment:
I don't think this warrants a Changelog entry. There hasn't been a release where this was broken